src/robusta/core/model/runner_config.py

```diff
@@ -32,3 +33,7 @@ class RunnerConfig(BaseModel):
     ]
     global_config: Optional[dict] = {}
     active_playbooks: Optional[List[PlaybookDefinition]] = []
+
+    @validator('global_config')
```

Comment:
It seems weird to do this with a validator, but I don't have any real objection to that.

Reply:
I feel the same, but think it's the best possible implementation